packages/aragon-wrapper/src/utils/abi.js

```diff
@@ -0,0 +1,26 @@
+export function findMethodAbiFragment (abi, methodSignature) {
+  if (methodSignature === 'fallback') {
+    return abi.find(method => method.type === 'fallback')
```

Comment:
Note for future: fallback functions do not contain `name` or `inputs` fields in the ABI.